README.md

```diff
@@ -1 +1,51 @@
-# To-do-list
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# # To-do-list
+
+In this project, we built a basic website about todo list 
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Download it as a ZIP file
+or
+
+### Clone it in your local machine using GIT
+to get a local copy , clone the repository using git clone
+(https://github.com/mr-abdellah/To-do-list.git)
```

Comment:
- [ ] Great job so far, after cloning the project there are some steps(installation) that we must follow in order to run the project` (npm install npm run build)` to name some, kindly list those steps so that it gives an idea of how to run the project. You got this 💪.